.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
```

Comment:
If you don't know it @olivierlsc  : You should try this  https://travis-ci.org

Reply:
actually the linter is not passing : https://travis-ci.org/xrobert35/swagger-express-ts